src/probability/kernel/cond_cdf.lean

```diff
@@ -0,0 +1,969 @@
+/-
+Copyright (c) 2023 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import measure_theory.constructions.borel_space
+import measure_theory.measure.stieltjes
+import probability.kernel.composition
+import measure_theory.decomposition.radon_nikodym
+
+/-!
+# Conditional cumulative distribution function
+
+Given `ρ : measure (α × ℝ)`, we define the conditional cumulative distribution function
```

Comment:
Maybe mention in this paragraph that you prove these results for finite rho. (Would it be enough that `ρ.fst` is sigma-finite, by the way?)

Reply:
sigma finite `ρ` would be enough, but I did not investigate the changes required yet. I don't know about sigma finite `ρ.fst`.
I think I wrote a TODO about that in the disintegration PR, but I forgot to copy it here.

Reply:
With sigma finite rho you can not take any more a cdf between 0 and 1 (what about Lebesgue measure on R x R, for instance? or Lebesgue times counting on R x Q?) Sigma finite `ρ.fst` looks to me as the right condition to make sure that you can keep a cdf from 0 to 1. In any case, it doesn't belong to this PR, which is already large enough (and impressive enough!)

Reply:
You're right. I got confused by the fact that we can obtain a disintegration for sigma-finite rho. But we do it by saying that a sigma finite rho is some transformation of a finite measure, and then we build the cond_cdf for that measure. That does not mean that we can get a conditional cdf for rho itself.